autobahn/src/main/java/io/crossbar/autobahn/wamp/Session.java

```diff
@@ -114,6 +114,13 @@ public Session(ExecutorService executor) {
         mExecutor = executor;
     }
 
+    public long getID() {
+        if (!isConnected()) {
```

Comment:
I'd remove the whole guard, and simply return null (mSessionID MUST be set to null when the session has left a realm already - and of course it must be null when the transport is detached then too)